packages/component/src/createLoadable.js

```diff
@@ -151,16 +151,19 @@ function createLoadable({ resolve = identity, render, onLoad }) {
       loadAsync() {
         if (!this.promise) {
           const { __chunkExtractor, forwardedRef, ...props } = this.props
-          this.promise = ctor
-            .requireAsync(props)
+
+          const cachedPromise = this.getCache()
+
+          this.promise = cachedPromise || ctor.requireAsync(props)
```

Comment:
What if the cache is a bit obsolete?
What if it was rejected by any reason? 

Now you are not able to restart the operation.

Reply:
I would make the promise retryable, instead of re-rendering from the error boundary :) although there might be use cases for the re-render, I'll think for a bit about this

Reply:
Promise cannot be retryable once it resolved or rejected - those are final states.
And in case of some error the normal expectation nowadays is to throw that error, and handle it at the nearest boundary.

Reply:
this was what I had in mind when I said "making" the promise retryable
```
retry = (fn, retries = 3) =>
    fn().catch(err => 
            retries > 1 
                ? retry(fn, retries - 1) 
                : Promise.reject(err))

loadable(() => retry(() => import('foo')))
```

Reply:
It's good to have "retry" working out of the box, but usually, it's better to expose this ability to the user, as long as they might want to communicate the problem to the end-user.